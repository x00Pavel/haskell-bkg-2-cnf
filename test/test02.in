S,A,B
a,b
S
S->aAB
A->BBB
S->BA
A->a
B->AS
B->b