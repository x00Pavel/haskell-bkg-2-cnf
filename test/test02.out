S,A,B,a',<AB>,<BB>
a,b
S
A->a
B->b
S->BA
B->AS
S->a'<AB>
a'->a
<AB>->AB
A->B<BB>
<BB>->BB